gallery/src/pageConfig.ts

```diff
@@ -183,7 +187,10 @@ const pageConfig: PageConfig = {
     'Status Indicator': { content: NxStatusIndicatorPage, type: 'html' },
     'Threat Indicator': { content: NxThreatIndicatorPage, type: 'react' },
     'Threat Indicator Legend': { content: NxThreatIndicatorLegendPage, type: 'react' },
+    'Vulnerability Indicator': { content: NxVulnerabilityIndicatorPage, type: 'react' },
+    'Vulnerability Indicator Legend': { content: NxVulnerabilityIndicatorLegendPage, type: 'react' },
     'Threat Number': { content: NxThreatNumberPage, type: 'html' },
+    'Vulnerability Score': { content: NxVulnerabilityScorePage, type: 'html' },
```

Comment:
Move all of the vuln components down to where they'd normally sort under the letter V

Reply:
Updated here: [0ea3129](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/0ea31298e43d965f4d28e9acd0e52459dbc87b44)

Reply:
> Updated here: [0ea3129](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/0ea31298e43d965f4d28e9acd0e52459dbc87b44)

Move them after "Overflow Tooltip"